dev/ory/kratos.yml

```diff
@@ -77,11 +77,23 @@ selfservice:
       after:
         password:
           hooks:
+            - hook: web_hook
+              config:
+                url: http://e2e-tests:4012/kratos/registration
+                method: POST
+                can_interrupt: true
+                body: file:///home/ory/body.jsonnet # TODO: use a base64 encoding instead
+                auth:
+                  type: api_key
+                  config:
+                    name: Authorization
+                    value: The-Value-of-My-Key
```

Comment:
we should create an unique key to authenticate the request. this key should also be passed to the api backend.